### PR TITLE
Move Behat dependencies to the test app

### LINKED ISSUE #11: Missing direct dependency
In this file: https://github.com/Sylius/Calendar/blob/main/config/services_test.yaml you define services that use Behat, but Behat is only a dev requirement. Therefore an application not using Behat will throw an exception like this when run in `test` env:

```
Invalid service "Sylius\Calendar\Tests\Behat\Context\Domain\CalendarContext": class "Behat\Behat\Context\Context" not found while loading "Sylius\Calendar\Tests\Behat\Context\Domain\CalendarContext".
```

I guess it's not the correct solution to move Behat to direct dependencies. Instead those test services should only be enabled if Behat is actually installed or some other solution?

### PR DESCRIPTION
**Pull request by coldic3**
Fixes https://github.com/Sylius/Calendar/issues/11

### CODE REVIEW

File: tests/Application/src/Kernel.php [hunk 1]
```diff
@@ -22,12 +25,27 @@ final class Kernel extends BaseKernel
...
+        $bundlesPath = $this->getProjectDir() . '/config/bundles.php';
         $contents = require $bundlesPath;
         foreach ($contents as $class => $envs) {
             if ($envs[$this->environment] ?? $envs['all'] ?? false) {
                 yield new $class();
             }
         }
     }
+
+    private function configureContainer(ContainerConfigurator $container, LoaderInterface $loader, ContainerBuilder $builder): void
```
Comment (GSadee) [inline tests/Application/src/Kernel.php]: I'm not sure if this configuration will resolve the initial problem described in the issue, such that an application not using Behat throws an exception when run in a test environment 🤔

Comment (coldic3) [reply tests/Application/src/Kernel.php]: I forgot to remove this method. It's copied from the `MicroKernelTrait` trait so it doesn't matter. To fix the issue I moved services declarations from `/config` to `/tests/Application`.

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Kevin! :1st_place_medal:

**Comment by loevgaard** [comment]
Will this PR be released soon? :)

**Comment by coldic3** [comment]
> Will this PR be released soon? :)

I think so, there are plans to release it this week.

**Pull request merged**